modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.install

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Hook implementations for OpenEuropa Bootstrap Theme Helper module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Component\Serialization\Yaml;
+
+/**
+ * Implements hook_install().
+ */
+function oe_bootstrap_theme_paragraphs_install() {
+  $config_files = [
+    'core.entity_form_display.paragraph.oe_links_block.default',
+    'core.entity_form_display.paragraph.oe_social_media_follow.default',
+    'core.entity_view_display.paragraph.oe_links_block.default',
+    'core.entity_view_display.paragraph.oe_social_media_follow.default',
+  ];
+
+  foreach ($config_files as $config_id) {
+    $raw_data = file_get_contents(drupal_get_path('module', 'oe_bootstrap_theme_paragraphs') . '/config/update/' . $config_id . '.yml');
```

Comment:
use FileStorage, see how it was done here https://github.com/openeuropa/oe_paragraphs/blob/56d401db70d49ec759c84cff39216cc86c2ecf6e/modules/oe_paragraphs_media/oe_paragraphs_media.post_update.php#L32